ipv6/ipv6.go

```diff
@@ -47,19 +48,30 @@ var _ = IPv6Describe("IPv6 Connectivity Tests", func() {
 		},
 	}
 
-	describeIPv6Tests := func(buildpackPath string, stack string) {
+	describeIPv6Tests := func(assetPath, stack string) {
 		appName = random_name.CATSRandomName("APP")
-		Expect(cf.Cf("push", appName,
-			"-m", DEFAULT_MEMORY_LIMIT,
-			"-p", buildpackPath,
-			"-s", stack,
-		).Wait(Config.DetectTimeoutDuration())).To(Exit(0))
+
+		if assetPath == "" {
```

Comment:
The checks for `assetPath == ""` look a bit odd. As the Java Spring app requires a different logic, can you try to move that coding into a separate function? E.g. `describeIPv6JavaSpring`?